File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/C/progam4.c

#include <stdio.h>
#include <stdlib.h>

int main () {

	int n, k;
	printf("Enter Size : ");
	scanf("%d" , &n);

	if (n < 0)
		exit(0);

	k = n * 2 - 1;
	for(int i = 1; i <= n; i++){
		for(int j = 1; j <= i; j++){
			printf("%d\t", k--);
		}
		k = n * 2 - 1 - i;
		printf("\n");
	}
}

File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/Java/progam4.java
import java.io.*;

class Template {
	public static void main(String[] args) throws IOException {
		
		int n = 0, k;
		System.out.printf("Enter Size : ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		try {
			n = Integer.parseInt(br.readLine());
		}catch(NumberFormatException nf){
			System.exit(0);
		}

		if(n <= 0)
			System.exit(0);

		k = n * 2 - 1;
		for(int i = 1; i <= n; i++){
			for(int j = 1; j <= i; j++){
				System.out.printf("%d\t", k--);
			}
			k = n * 2 - 1 - i;
			System.out.printf("\n");
		}
	}
}